software/control/widgets.py

```diff
@@ -1997,348 +1997,30 @@ def display_stats(self, stats):
             row+=1
 
 
-class MultiPointWidget(QFrame):
+class FlexibleMultiPointWidget(QFrame):
```

Comment:
Can you switch the order of `WellPlateMultiPointWidget` and `FlexibleMultiPointWidget`? Current order deletes a lot of code and re-add a lot of code. Please try to avoid this kind of scenario.

Reply:
this is the old multipoint widget